.github/PULL_REQUEST_TEMPLATE/PULL_REQUEST_TEMPLATE.yml

```diff
@@ -0,0 +1,20 @@
+name: Pull Request
+description: Create a pull request to propose and collaborate on changes to this repository.
+labels:
+  - suggestion
```

Comment:
What does that mean? Where would it be added?

Reply:
removed